test/mpi/.gitignore

```diff
@@ -13,23 +10,18 @@
 /attr/attrorder
 /attr/attrordercomm
 /attr/attrordertype
-/attr/attrorderwin
 /attr/attrt
 /attr/baseattr2
 /attr/baseattrcomm
-/attr/baseattrwin
 /attr/fkeyval
 /attr/fkeyvalcomm
 /attr/fkeyvaltype
-/attr/fkeyvalwin
 /attr/keyval_double_free
 /attr/keyval_double_free_comm
 /attr/keyval_double_free_type
 /attr/keyval_double_free_win
 /attr/testlist
-/basic/Netpipe.out
```

Comment:
this one is valid, too. it is a temp file created by `test/mpi/basic/netmpi.c`

Reply:
it looks like this entry was removed in this commit, and then added back in the sorting commit. rather than nitpick on this, can we just squash these 2 commits together before merge?